BYT/ColorScheme.swift

```diff
@@ -94,8 +98,8 @@ class ColorScheme {
             let version_availableArrayOfDict = dict["version_available"] as? [[String: Any]]{
             var v = [VersionInColorScheme]()
             for dict in version_availableArrayOfDict {
```

Comment:
this `var v` and `dict` aren't used, resulting in compiler warning. can this be removed?